File: Examples/Output/arrays_length_test.java
class ArraysLengthDemo {
    public static void main(String[] args) {
        int[] valores = new int[] { 4, 7, 9, 12 };
        int suma = 0;
        for (int i = 0; i < valores.length; i++) {
            suma = (suma + valores[i]);
        }
        boolean largo = (valores.length > 3);
        System.out.println(suma);
        System.out.println(largo);
    }
}

File: Examples/Input/arrays_length_test.cs
using System;

class ArraysLengthDemo
{
    public static void Main(string[] args)
    {
        int[] valores = { 4, 7, 9, 12 };
        int suma = 0;

        for (int i = 0; i < valores.Length; i++)
        {
            suma = suma + valores[i];
        }

        bool largo = valores.Length > 3;

        Console.WriteLine(suma);
        Console.WriteLine(largo);
    }
}
